configure.ac

```diff
@@ -2511,14 +2511,6 @@ if test "$WINDOWS_PATH" = "yes"; then
     e_logfilesdir="$e_logdir\\\\files"
     e_logcertsdir="$e_logdir\\\\certs"
     e_datarulesdir="$e_winbase\\\\rules\\\\"
-    if test "$HAVE_CYGPATH" != "no"; then
-        # turn srcdir into abs path and convert to the
-        # mixed output (/c/Users/dev into  c:/Users/dev)
-        e_rustdir="$(cygpath -a -t mixed ${srcdir})/rust"
```

Comment:
Rust will break without this path conversion trickery, as you see in the appveyor errors
```
Compiling suricata v5.0.0-dev (C:\projects\suricata\rust)
error: couldn't read C:/c/projects/suricata/rust/src/lib.rs: The system cannot find the path specified. (os error 3)
```